docs/csharp/programming-guide/statements-expressions-operators/expression-bodied-members.md

```diff
@@ -1,100 +1,92 @@
+Support for expression body definitions was introduced for methods and read-only properties in C# 6 and was expanded in C# 7.0. Expression body definitions can be used with the type members listed in the following table:
 
 |Member  |Supported as of... |
 |---------|---------|
 |[Method](#methods)  |C# 6 |
+|[Read-only property](#read-only-properties)   |C# 6  |
+|[Property](#properties)  |C# 7.0 |
 |[Constructor](#constructors)   |C# 7.0 |
 |[Finalizer](#finalizers)     |C# 7.0 |
-|[Property Get](#property-get-statements)  |C# 6 |
-|[Property Set](#property-set-statements)  |C# 7.0 |
 |[Indexer](#indexers)       |C# 7.0 |
 
 ## Methods
 
-An expression-bodied method consists of a single expression that returns a value whose type matches the method's return type, or, for methods that return `void`, that performs some operation. For example, types that override the <xref:System.Object.ToString%2A> method typically include a single expression that returns the string representation of the current object. 
+An expression-bodied method consists of a single expression that returns a value whose type matches the method's return type, or, for methods that return `void`, that performs some operation. For example, types that override the <xref:System.Object.ToString%2A> method typically include a single expression that returns the string representation of the current object.
 
 The following example defines a `Person` class that overrides the <xref:System.Object.ToString%2A> method with an expression body definition. It also defines a `DisplayName` method that displays a name to the console. Note that the `return` keyword is not used in the `ToString` expression body definition.
 
 [!code-csharp[expression-bodied-methods](../../../../samples/snippets/csharp/programming-guide/classes-and-structs/expr-bodied-methods.cs)]  
 
 For more information, see [Methods (C# Programming Guide)](../classes-and-structs/methods.md).
- 
-## Constructors
 
-An expression body definition for a constructor typically consists of a single assignment expression or a method call that handles the constructor's arguments or initializes instance state. 
-
-The following example defines a `Location` class whose constructor has a single string parameter named *name*. The expression body definition assigns the argument to the `Name` property.
+## Read-only properties
 
-[!code-csharp[expression-bodied-constructor](../../../../samples/snippets/csharp/programming-guide/classes-and-structs/expr-bodied-ctor.cs#1)]  
-
-For more information, see [Constructors (C# Programming Guide)](../classes-and-structs/constructors.md).
+Starting with C# 6, you can use expression body definition to implement a read-only property. To do that, use the following syntax:
 
-## Finalizers
+```csharp
+PropertyName => expression;
```

Comment:
I think this would be more clear if it were:

```csharp
PropertyType PropertyName => expression;
```

Thoughts?

Reply:
@BillWagner I agree with you; updated. Thanks!